keycloak-iam/src/main/java/com/valtimo/keycloak/security/jwt/authentication/KeycloakTokenAuthenticator.java

```diff
@@ -50,19 +50,19 @@ public KeycloakTokenAuthenticator(String keycloakClient) {
     public boolean supports(Claims claims) {
         try {
             final String email = getEmail(claims);
-            if (email.isBlank()) {
-                logger.debug("Support failed: email is blank");
+            if (email == null || email.isBlank()) {
```

Comment:
I think this brings up an interesting point. If the value is there but empty, you can argue it's deliberate (and so we can say the email should not be blank), but if the field is not there altogether, then email is not left blank, because it isn't present. Maybe this warrants a separate if statement instead, stating "email must be present"?

Reply:
Agreed, I can make the null check a separate if statement